pkg/ctl/utils/enable_logging.go

```diff
@@ -0,0 +1,110 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/printers"
+)
+
+func enableLoggingCmd(rc *cmdutils.ResourceCmd) {
```

Comment:
Can we add a unit test for this since it has some logic?

Reply:
We have it covered by integration suite now. Do you insist on unit test as well?

Reply:
I absolutely agree, but I won't have the time this week.